examples/mutag_hdt.py

```diff
@@ -0,0 +1,214 @@
+from pyrdf2vec.connectors import Connector
+import matplotlib.pyplot as plt
+import pandas as pd
+from sklearn.manifold import TSNE
+
+from pyrdf2vec import RDF2VecTransformer
+from pyrdf2vec.embedders import Word2Vec
+from pyrdf2vec.graphs import KG
+from pyrdf2vec.walkers import RandomWalker
+from pyrdf2vec.typings import Literal, Response
+from cachetools import cachedmethod
+from cachetools.keys import hashkey
+import operator
+from functools import partial
+from typing import List, Optional
+from rdflib import URIRef, Literal
+import rdflib_hdt
+import attr
+from sklearn.model_selection import GridSearchCV
+from sklearn.svm import SVC
+from sklearn.metrics import accuracy_score, confusion_matrix
+
+# Ensure the determinism of this script by initializing a pseudo-random number.
+RANDOM_STATE = 22
+
+pos_entities = ["http://dl-learner.org/benchmark/dataset/animals#dog01",
+            "http://dl-learner.org/benchmark/dataset/animals#dolphin01",
+            "http://dl-learner.org/benchmark/dataset/animals#platypus01",
+            "http://dl-learner.org/benchmark/dataset/animals#bat01"]
+
+neg_entities = ["http://dl-learner.org/benchmark/dataset/animals#trout01",
+            "http://dl-learner.org/benchmark/dataset/animals#herring01",
+            "http://dl-learner.org/benchmark/dataset/animals#shark01",
+            "http://dl-learner.org/benchmark/dataset/animals#lizard01",
+            "http://dl-learner.org/benchmark/dataset/animals#croco01",
+            "http://dl-learner.org/benchmark/dataset/animals#trex01",
+            "http://dl-learner.org/benchmark/dataset/animals#turtle01",
+            "http://dl-learner.org/benchmark/dataset/animals#eagle01",
+            "http://dl-learner.org/benchmark/dataset/animals#ostrich01",
+            "http://dl-learner.org/benchmark/dataset/animals#penguin01"]
+
+train_entities = [entity for entity in pos_entities[0:2]+neg_entities[0:5]]
+train_labels = list([1]*2+[0]*5)
+
+test_entities = [entity for entity in pos_entities[2:]+neg_entities[5:]]
+test_labels = list([1]*2+[0]*5)
+
+entities = train_entities + test_entities
+labels = train_labels + test_labels
+
+
+@attr.s
+class HDTConnector(Connector):
+    def __attrs_post_init__(self):
+        self.store = None
+
+    @cachedmethod(operator.attrgetter("cache"), key=partial(hashkey, "fetch"))
```

Comment:
@bsteenwi Could you remove `key=partial(hashkey, "fetch")`? With the new version of `cachetool`, we have an issue by specifying the key.

`@cachedmethod(operator.attrgetter("cache"))` should be fine.